conf/app.routes

```diff
@@ -25,3 +25,5 @@ GET           /test-only/requests
 DELETE        /test-only/request/:requestId                                                                 uk.gov.hmrc.debttransformationstub.controllers.TimeToPayTestController.deleteTTPRequest(requestId: String)
 GET           /test-only/request/:requestId                                                                 uk.gov.hmrc.debttransformationstub.controllers.TimeToPayTestController.getTTPRequest(requestId: String )
 POST          /test-only/response                                                                           uk.gov.hmrc.debttransformationstub.controllers.TimeToPayTestController.createTTPRequests
+
+POST          /individuals/debts/field-collections/templates                                                uk.gov.hmrc.debttransformationstub.controllers.DebtManagementAPITemplatesTestController.fieldCollectionsTemplates()
```

Comment:
should this be in /test-only/... format?

Reply:
Right now the routes in the `test-only` format are the routes for the bridge tool, the "normal" stub routes follow this format I think. Maybe move it above the bridge tool routes?

Reply:
Moved url to DM level to differentiate from bridge tool url's